backend/inspirehep/search/aggregations.py

```diff
@@ -86,7 +88,11 @@ def hep_author_aggregation(order, author=None, title="Author", agg_type="checkbo
 def hep_subject_aggregation(order, title="Subject", agg_type="checkbox"):
     return {
         "subject": {
-            "terms": {"field": "facet_inspire_categories", "size": 20},
+            "terms": {
+                "field": "facet_inspire_categories",
+                "missing": current_app.config.get("SUBJECT_MISSING_VALUE", "Unknown"),
```

Comment:
Same, you already assigned `Unknown` to `SUBJECT_MISSING_VALUE` - is there any possibility that it would change?